README.md

```diff
@@ -1,7 +1,11 @@
 ## Configure
 ```
+### From source code
 sudo python3 -m pip install pipenv 
-alias standup-notes=<workspace>/standup_notes/bin/standup-notes
+sudo python3 setup.py install
```

Comment:
Looks like you might be able to just do the install for the user:
```
python3 setup.py install --user
```
Side benefit, it doesn't create directories (_build/_, _dist/_, _*egg/_) in the source directory.